hoomd/pytest/test_version.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2009-2020 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+import hoomd
+
+def test_version():
+    """Test version information."""
+    assert hoomd.__version__
```

Comment:
It might be possible to add other version tests here, such as parsing with `packaging.version.parse()` or checking some assertions about the other flags/info in `hoomd.version`. I think the `packaging` test would require adding `packaging` as a dependency. Please comment if you have ideas for what to test here.